isofit/utils/template_construction.py

```diff
@@ -561,7 +576,7 @@ def build_presolve_config(
     if emulator_base is not None:
         radiative_transfer_config["radiative_transfer_engines"]["vswir"][
             "emulator_file"
-        ] = abspath(emulator_base)
+        ] = abspath(emulator_base) + ".h5" # sc - can't find emulator file without ".h5" in the name
```

Comment:
@skyecaplan What was the reason for this modification? If you point to the .h5 file directly with the `emulator_base` argument, you shouldn't get an error here. Adding another ".h5" would lead to a failure if users provide `.../sRTMnet/sRTMnet_v120.h5 `as input.